components/app/project-detail/ProjectSingleLink.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <tr>
+    <td class="flex my-2">
+      <img class="mr-2" :src="icon" :alt="title + ' ' + $t('general.icon')" />
```

Comment:
thought (non-blocking): consider using a computed property here to clean up the code.

Reply:
ProjectSingleLink can be deleted in the future after the commented section 'jobs' has been outsourced from ProjectSingle page.